user_sync/app.py

```diff
@@ -367,32 +367,55 @@ def credentials():
     pass
 
 
-@credentials.command(
-    help="Stores all sensitive fields and updates configuration files, replacing plaintext values with keys")
+@credentials.command(help="Stores all sensitive fields and updates configuration files, replacing plaintext values with keys")
+
 def store():
     """
-    Stores credentials in the configuration file
+    Stores secure credentials in the configuration file
     This is an automated process
     """
+
     credential_manager = CredentialManager()
-    click.echo("Using backend: " + credential_manager.keyring_name)
-    click.echo("You have called the store-credential command.")
+    credential_manager.store()
+    retrieve_from_credman_keys = credential_manager.retrieve_from_credman_keys()
+    retrieve_config_credentials = credential_manager.retrieve_config_credentials()
+    if retrieve_config_credentials != retrieve_from_credman_keys:
+        raise AssertionException("Values were not stored securely in configuration file. Stored values are:"
+                                 + dict(retrieve_config_credentials))
+    click.echo(f"{retrieve_config_credentials} stored securely in configuration file.")
 
 
-@credentials.command(help="Retreives currently stored credentials under the username 'user_sync'.")
-@click.option('--revert', default=False, is_flag=True, help="Reverts configuration files to plaintext state.")
-def retrieve(revert):
+@credentials.command(help="Retrieves currently stored credentials under the username 'user_sync'.")
+def retrieve():
     """
     Retrieves credentials from the configuration files
     By default, just prints out the values stored for UST
-    If --revert is used, updates config files with actual plaintext data
-    This is an automated process
     """
     credential_manager = CredentialManager()
-    click.echo("Using backend: " + credential_manager.keyring_name)
-    if revert:
-        click.echo("reverting...")
-    click.echo("you have called the retrieve credential command")
+
+    retrieve_from_credman_keys = credential_manager.retrieve_from_credman_keys()
+
+    if retrieve_from_credman_keys is None:
+        raise AssertionException("Unable to retrieve credentials")
+    click.echo(f"The following values were stored securely: {retrieve_from_credman_keys}")
+
+
+@credentials.command(help="Will return configuration file to unsecured state and replace all secrure values with "
+                          "plain text values.")
+def revert():
+    """
+    Revert updates config files
+    with actual plaintext data This is an
+    automated process.
+    """
+    credential_manager = CredentialManager()
+    retrieve_from_credman_txt = credential_manager.retrieve_from_credman_txt()
+    credential_manager.revert()
+    retrieve_config_credentials = credential_manager.retrieve_config_credentials()
+    if retrieve_config_credentials != retrieve_from_credman_txt:
```

Comment:
Same goes here - if there is an error during the process, let credential manager raise it, and just simply handle them here